IPython/utils/path.py

```diff
@@ -98,18 +99,13 @@ def get_py_filename(name, force_win32=None):
 
     If the given name is not a file, it adds '.py' and searches again.
     Raises IOError with an informative message if the file isn't found.
-
-    On Windows, apply Windows semantics to the filename. In particular, remove
-    any quoting that has been applied to it. This option can be forced for
-    testing purposes.
     """
 
     name = os.path.expanduser(name)
-    if force_win32 is None:
-        win32 = (sys.platform == 'win32')
-    else:
-        win32 = force_win32
-    name = unquote_filename(name, win32=win32)
+    if force_win32 is not None:
```

Comment:
Haha ! But someone could be passing `None` explicitly and will not see the warning !

Just kidding I think it's fine as is !